waydroid-patches/apply-patches.sh

```diff
@@ -69,6 +69,7 @@ apply_patch() {
       echo -e ${reset}""${reset} 
       cd $top_dir
       project_revision=`repo info $current_project | grep 'Current revision: ' | sed 's/Current revision: //'`
+      project_log=`git log --max-count=1000 --pretty="format:%aD, %s" $project_revision..`
```

Comment:
This runs from $top_dir, so it needs to chdir into the git project first

```suggestion
      project_log=`git -C $current_project log --max-count=1000 --pretty="format:%aD, %s" $project_revision..`
```